README.md

```diff
@@ -17,63 +17,137 @@ julia> y = collect(1:100) + randn(100)*10;
 julia> X = [1:100 (1:100)+randn(100)*5 (1:100)+randn(100)*10 (1:100)+randn(100)*20];
 
 julia> path = glmnet(X, y)
-Least Squares GLMNet Solution Path (55 solutions for 4 predictors in 163 passes):
-55x3 DataFrame:
-         df  pct_dev        λ
-[1,]      0      0.0  27.1988
-[2,]      1 0.154843  24.7825
-[3,]      1 0.283396  22.5809
-  :
-[53,]     2 0.911956 0.215546
-[54,]     2 0.911966 0.196397
-[55,]     2 0.911974  0.17895
+Least Squares GLMNet Solution Path (74 solutions for 4 predictors in 832 passes):
```

Comment:
this would be easier if we had a Documenter.jl instead of a Readme file, though that's probably more overhead than either of us want/need to put in. Be that as it may, I went through and actually ran the code and made a commit with the new output. I also rewrote the plotting example so that it works with Plots.jl (no recipe, just a prototype plot in the documentation).

The file readmeExamples.jl has the actual code run that should be reproducible.